lib/commanded/commands/decriber.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Commanded.Commands.Describer do
+  @moduledoc """
+  Commands describer is used to describe a command, mostly for filtering sensitive data from being exposed in log outputs.
+
+  ## Example
+      config :commanded, :filter_fields, [:password, :secret_key]
+  """
+
+  @fields_to_filter Application.get_env(:commanded, :filter_fields, [])
```

Comment:
The `Application.get_env(:commanded, :filter_fields, [])` call will need to be done at runtime, not compile time, or else changing the config will not have an effect until commanded dependency is recompiled (e.g. `mix deps.compile --force command`).